index.html

```diff
@@ -45,4 +51,4 @@ <h1>Note Editor!</h1>
         }
         </script>
     </body>
-</html>
+</html>
```

Comment:
There should usually be a newline at the end of files, as Github is pointing out right here.

Reply:
Updated!

Reply:
Thank you very much!